packages/smarthr-ui/src/components/Table/EmptyTableBody.tsx

```diff
@@ -93,7 +93,7 @@ export const EmptyTableBody: React.FC<Props & ElementProps> = ({ children, paddi
     <tbody {...props} ref={countHeadCellRef}>
       <tr>
         <Td colSpan={count} className={tdStyles}>
-          <Center>{children}</Center>
+          <Center className="shr-text-center">{children}</Center>
```

Comment:
`<Center />` に特に何も渡していないので、ただ消す方がよさそうに見えました!

```suggestion
          {children}
```

Reply:
たしかに！修正しました [ffd34eb](https://github.com/kufu/smarthr-ui/pull/5186/commits/ffd34ebb44afb79b8e880b361167ff2035d5c4bc)